typegate/native/Cargo.toml

```diff
@@ -50,7 +50,11 @@ rust-s3 = "0.33.0"
 http = "0.2.9"
 envconfig = "0.10.0"
 once_cell = "1.18.0"
-wasmedge-sdk = { version = "0.8.1", features = ["standalone"] }
 base64 = "0.21.2"
 tap = "1.0.1"
+num-traits = "0.2"
+num-derive = "0.3"
+wasmedge-sdk = { version = "0.8.1", features = ["standalone"] }
```

Comment:
can you add a comment here why 0.9.0 did not work?

Reply:
1. Vm -> Vm<T>: we have to update wasmedge_sdk_bindgen
2. After some cleanups, even loading the reactor simply did not work (no panic, nothing) the program just quit with exit code 11.